clients/cloud/clojure/README.md

```diff
@@ -0,0 +1,72 @@
+# Clojure Producer and Consumer for Confluent Cloud
+
+This directory includes projects demonstrating how to use Java interop
+form Clojure to produce and consume with Confluent Cloud.
+
+For more information, please see the [application development documentation](https://docs.confluent.io/current/api-javadoc.html)
+
+# Prerequisites
+
+* Installed [Leiningen](https://leiningen.org/#install) tool
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Initialize a properties file at `$HOME/.ccloud/config` with configuration to your Confluent Cloud cluster:
+
+```shell
+$ cat $HOME/.ccloud/config
+bootstrap.servers=<BROKER ENDPOINT>
+ssl.endpoint.identification.algorithm=https
+security.protocol=SASL_SSL
+sasl.mechanism=PLAIN
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<API KEY>" password\="<API SECRET>";
+
+# Quickstart
```

Comment:
All the text in the README should be an exact replica of the other languages in this folder.  Can you please modify all the text to match?